Unable to < make repl >

Hi Clojerl team,

For some reason I'm unable to simply `make repl` 

Here's my error log 

```
rlwrap: error:
  prompt colour spec should be <attr>;<fg>[;<bg>]
  where <attr> ranges over [0...8], <fg> over [30...37] and <bg> over [40...47]
  example: 0;33 for yellow on current background, 1;31;40 for bold red on black
Makefile:34: recipe for target 'repl' failed
make: *** [repl] Error 1

```

I'm on Ubuntu16.04 and relying on `nix` package manager for the `erlang` and `rebar3` dependencies

--- a/Makefile
+++ b/Makefile
@@ -3,6 +3,7 @@
 # Show less verbose output
 .SILENT:
 
+SHELL     := /bin/bash
 REBAR3    := rebar3
 RLWRAP    := $(shell type -p rlwrap &> /dev/null && echo rlwrap || echo)
 V         := @

--- a/bin/clje
+++ b/bin/clje
@@ -1,7 +1,20 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
+
+readlink_f () {
+  cd "$(dirname "$1")" > /dev/null
+  filename="$(basename "$1")"
+  if [ -h "$filename" ]; then
+    readlink_f "$(readlink "$filename")"
+  else
+    echo "`pwd -P`/$filename"
+  fi
+}
+
+SELF=$(readlink_f "$0")
+CLJE_BINDIR=$(dirname "$SELF")
 
 if type -p rlwrap >/dev/null 2>&1; then
-  rlwrap -r -q '\"' -b "(){}[],^%3@\";:'" clojerl "$@"
+  rlwrap -r -q '\"' -b "(){}[],^%3@\";:'" $CLJE_BINDIR/clojerl "$@"
 else
   echo "Please install rlwrap for command editing or use \"clojerl\" instead."
 fi

